package.json

```diff
@@ -38,13 +38,15 @@
     "dependencies": {
         "@types/cli-spinner": "^0.2.0",
         "@types/node": "^9.6.55",
+        "@types/prompts": "^2.0.3",
         "@types/readline-sync": "^1.4.3",
         "chalk": "^2.3.0",
         "cli-spinner": "^0.2.7",
         "cli-table3": "^0.5.1",
         "clime": "^0.5.9",
         "nem2-sdk": "^0.16.3",
         "path": "^0.12.7",
+        "prompts": "^2.1.0",
```

Comment:
Add promts license here: https://github.com/nemtech/nem2-cli/blob/master/LICENSE#L207 and under ``/licenses`` folder.